tests/composer/auto_complete/function_auto_complete.test.ts

```diff
@@ -0,0 +1,18 @@
+import { ComposerStore } from "../../../src/components/composer/composer/composer_store";
+import { setCellContent } from "../../test_helpers/commands_helpers";
+import { makeStore } from "../../test_helpers/stores";
+
+describe("Function auto complete", () => {
+  test("start with exact match, but with other proposals", () => {
```

Comment:
File nae should be appended with `_store`. Same for `sheet_name_auto_complete.test.ts`